subsys/bluetooth/controller/ll_sw/lll_conn_iso.h

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (c) 2021 Demant
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+struct node_tx_iso {
+	union {
+		void        *next;
+		memq_link_t *link;
+	};
+
+	uint64_t payload_number : 39; /* cisPayloadNumber */
+	uint8_t  pdu[];
+};
+
+struct lll_conn_iso_strm_rxtx {
+	uint8_t phy;            /* PHY */
+	uint8_t burst_number;   /* Burst number (BN) */
+	uint8_t flush_timeout;  /* Flush timeout (FT) */
+	uint8_t max_octets;     /* Maximum PDU size */
+};
+
+struct lll_conn_iso_strm {
+	/* Link to ACL connection (for encryption, channel map, crc init) */
+	struct lll_conn *conn;
+
+	/* Connection parameters */
+	uint8_t  access_addr[4];    /* Access address */
+	uint32_t offset;            /* Offset of CIS from start of CIG in us */
+	uint32_t sub_interval;      /* Interval between subevents in us */
+	uint32_t subevent_length;   /* Length of subevent in us */
+	uint8_t  num_subevents;     /* Number of subevents */
+	struct lll_conn_iso_strm_rxtx rx; /* RX parameters */
+	struct lll_conn_iso_strm_rxtx tx; /* TX parameters */
+
+	/* Event and payload counters */
+	uint64_t event_count : 39;       /* cisEventCount */
+	uint64_t rx_payload_number : 39; /* cisPayloadNumber */
+
+	/* Acknowledgment and flow control */
+	uint8_t sn:1;               /* Sequence number */
+	uint8_t nesn:1;             /* Next expected sequence number */
+	uint8_t cie:1;              /* Close isochronous event */
+
+	/* Resumption information */
+	uint8_t next_subevent;      /* Next subevent to schedule */
+
+	/* Transmission queue */
+	MEMQ_DECLARE(tx);
+	memq_link_t link_tx;
+	memq_link_t *link_tx_free;
+};
+
+struct lll_conn_iso {
+	struct lll_hdr hdr;
+
+	uint8_t  num_cis;  /* Number of CISes in this CIG */
+#if defined(CONFIG_BT_CTLR_CONN_ISO_STREAMS)
+	struct lll_conn_iso_strm strms[CONFIG_BT_CTLR_CONN_ISO_STREAMS];
+#endif /* CONFIG_BT_CTLR_CONN_ISO_STREAMS */
```

Comment:
Do we need the guard here? 

`CONFIG_BT_CTLR_CONN_ISO_STREAMS` doesn't seem to be guarded by anything in the Kconfig, so it should always be visible/defined.

Reply:
`CONFIG_BT_CTLR_CONN_ISO_STREAMS` has a `depends on BT_CTLR_CENTRAL_ISO || BT_CTLR_PERIPHERAL_ISO`, so some guard is needed.

Reply:
Oh wow, I missed that somehow :s